bin/restart

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift File.expand_path('../../', __FILE__)
+
+require 'config/initializers/figaro'
+require 'winrm'
+
+options = {
+  endpoint: format(
+    'http://%s:%s/wsman',
```

Comment:
I didn't try, I will give it a go... worse case I can add to env for those configurations that are using `https` as I believe it should be supported, not sure if our servers are set up for it though.